src/Native/System.Native/pal_process.cpp

```diff
@@ -417,3 +427,51 @@ extern "C" char* GetCwd(char* buffer, int32_t bufferSize)
 
     return getcwd(buffer, UnsignedCast(bufferSize));
 }
+
+#if HAVE_SCHED_SETAFFINITY
+static void ConvertPalToNativeCpuSet(const CpuSetBits& pal, cpu_set_t& native)
+{
+    memcpy(native.__bits, pal.Bits, ARRAY_SIZE(native.__bits));
```

Comment:
Earlier we statically asserted that sizeof(CpuSetBits) >= sizeof(cpu_set_t).  So while unlikely, it's possible that CpuSetBits is larger than cpu_set_t, which means it's possible that this call could be trying to set affinity to a value that's out of range for the native side, and because we're only coping ARRAY_SIZE(native.__bits), we'll end up trimming those off and losing the information.  Are we ok with that?